conf/mini_provisioner/v2/ha.config.tmpl.3-node

```diff
@@ -24,6 +24,10 @@ server_node:
     storage_set_id: TMPL_STORAGE_SET_ID_1
     s3_instances: '1'
     type: TMPL_TYPE
+    bmc:
+      ip: TMPL_BMC_IP
```

Comment:
Please modify to TMPL_BMC_IP_1, TMPL_BMC_IP_2, TMPL_BMC_IP_3, etc. for the 3-node files. Please update user and secret too.